cypress/integration/chi-js/enterprise-global-nav.spec.js

```diff
@@ -0,0 +1,395 @@
+describe('enterprise-nav-menu', function() {
+  before(() => {
+    cy.visit('http://localhost:8000/tests/js/enterprise-global-nav.html');
+  });
+
+  describe('check that click on 1st level opens 2nd-level menu drawer', () => {
+    [/^NoChanges$/, /^Changes$/, /^DeepLink$/].forEach((menuText) => {
+      it('check "' + menuText + '" opens 2nd-level menu drawer', () => {
+
+        cy.get('.enterprise-nav-menu').within(() => {
```

Comment:
Please use constants *if possible* for selectors. Try to import them directly from component project, so if it changes, you don't have to change the test. 

The best practice is to use custom selectors for tests (i.e. data-cy="enterprise-nav-menu"), but I think not for Chi, as it will increase component verbosity. So, at least, please use constants.